runtime/shibuya/src/lib.rs

```diff
@@ -1311,10 +1312,44 @@ pub type Executive = frame_executive::Executive<
     Migrations,
 >;
 
+pub use frame_support::traits::{OnRuntimeUpgrade, StorageVersion};
+pub struct HybridInflationModelMigration;
+impl OnRuntimeUpgrade for HybridInflationModelMigration {
+    fn on_runtime_upgrade() -> Weight {
+        let mut reward_config = block_rewards_hybrid::RewardDistributionConfig {
+            // 4.66%
+            treasury_percent: Perbill::from_rational(4663701u32, 100000000u32),
```

Comment:
Please update to format like `4_663_701u32` for readability and make it easier to catch mistakes.